src/core/bittorrent/torrenthandle.h

```diff
@@ -370,7 +372,10 @@ namespace BitTorrent
         // m_queuedPath is where files should be moved to,
         // when current moving is completed
         QString m_queuedPath;
-        QQueue<MoveStorageTrigger> m_moveStorageTriggers;
+        // m_moveFinishedTriggers is activated only when the following conditions are met:
+        // torrent finished, all file rename jobs complete, all file move jobs complete
```

Comment:
Torrent finished is a *condition* for the trigger to queue event and later activate. I'll remove it since you find it confusing...